packages/insomnia-app/app/ui/components/request-pane.js

```diff
@@ -82,6 +92,17 @@ class RequestPane extends React.PureComponent<Props> {
     });
   }
 
+  async _handlePluginTabs() {
+    getRequestTabs().then(
+      pluginTabs => this.setState({ pluginTabs }),
+      err => console.error(err),
+    );
```

Comment:
I think preference here would be to use try/catch with async/await to keep consistent with the rest of the app. Also proposed change to the message in console logs.

```suggestion
    try {
      const pluginTabs = await getRequestTabs();
      this.setState({ pluginTabs });
    } catch (err) {
      console.error('[plugins] Error getting request tabs', err)
	}
```

Reply:
fixed in d53beaf8